manifests/instance.pp

```diff
@@ -306,6 +310,7 @@
   elasticsearch::service { $name:
     ensure             => $ensure,
     status             => $status,
+    service_flags      => $init_flags,
```

Comment:
indeed, I initially had this as $init_flags, because of all the other init_* variables, but then, since the others don't really make sense that much, I changed it to service_flags, and seem to have this one forgotten, fixed now.